combat-trainer.lic

```diff
@@ -3077,6 +3129,17 @@ class GameState
     @construct_mode
   end
 
+  def determine_aiming_skill
+      held_types = ['Small Edged','Large Edged', 'Small Blunt','Large Blunt', 'Staves', 'Polearms']
+      options = @aiming_trainables
+      options = options - held_types if weapon_skill.eql?'Bow' or weapon_skill.eql?'Crossbow'
```

Comment:
💡 `options = options - held_types` could be `options -= held_types`

Reply:
Ah, yeah, I do this in my other PR. Just spaced, thanks!